### ISSUE TITLE
CMD click

### ISSUE THREAD

**Issue by stephenjason89** [fumeapp/nuxt-storm#10]
CMD click or alt click for windows redirects me to the .component.gen.js file instead of the original file.
Can I force it to directly open the original file??

**Comment by fabien-lopes** [comment]
@acidjazz +1

**Comment by acidjazz** [comment]
@stephenjason89 @fabien-lopes does this happen 100% of the time? I seem to see it happen only 10% of the time for me.

**Comment by stephenjason89** [comment]
@acidjazz It actually depends. If the component is natively detected by webstorm even without the nuxt-storm plugin, then CMD click opens the file itself. But if it is a component that webstorm wasn't able to detect, then it opens the component.gen.js file

**Comment by acidjazz** [comment]
@stephenjason89 @fabien-lopes when you click into `components.gen.js` does clicking the component from there work?

**Comment by fabien-lopes** [comment]
@acidjazz Yes, when I click on the component I redirected directly to the components.gen.js file on the line of the the component. From there, I can click on the component to be redirected to his page.

**Comment by stephenjason89** [comment]
> @stephenjason89 @fabien-lopes when you click into `components.gen.js` does clicking the component from there work?

@acidjazz oh yes. When at components.gen.js, i can click the link and be redirected to the original component.

**Comment by fabien-lopes** [comment]
Hello @acidjazz, any news ?
Many thanks for your help

**Comment by stephenjason89** [comment]
@acidjazz Would also want to gently follow up on this matter.

Thank you

**Comment by stephenjason89** [comment]
hey @fabien-lopes 
I found a solution to this.
Add a component name and everything will work perfectly.
It's just not so nice to add a component name on script setup tag because you'll need another script just for the name
